packages/web-components/src/spinner/spinner.options.ts

```diff
@@ -0,0 +1,37 @@
+import { ValuesOf } from '@microsoft/fast-foundation';
+
+export type Values<T> = T[keyof T];
```

Comment:
@procload I still see this, can we remove is it's duplicative of the above import?

```suggestion
```

Reply:
Doh! Will do.